modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -44,33 +44,51 @@ spec:
 ----
 <1> If you add this annotation to your NetworkAttachmentDefinition, your virtual machine instances
 will only run on nodes that have the `br0` bridge connected.
-<2> Required. A name for the configuration.
+<2> Required. A name for the configuration. It is recommended to match the configuration name to the `name` value of the NetworkAttachmentDefinition.
 <3> The actual name of the Container Network Interface (CNI) plug-in that provides
 the network for this NetworkAttachmentDefinition. Do not change this field unless
 you want to use a different CNI.
 <4> You must substitute the actual name of the bridge, if it is not `br0`.
 <5> Required. This allows the MAC pool manager to assign a unique MAC address to the connection.
+
 +
 ----
 $ oc create -f <resource_spec.yaml>
 ----
 
-. Edit the configuration file of a virtual machine or virtual machine instance that you want to connect to the
-bridge network:
+. Edit the configuration file of a virtual machine or virtual machine instance that you want to connect to the bridge network.
 +
 [source,yaml]
 ----
 apiVersion: v1
 kind: VirtualMachine
 metadata:
-  name: example-vm
-  annotations:
-    k8s.v1.cni.cncf.io/networks: a-bridge-network <1>
+    name: example-vm
 spec:
+...
+  interfaces:
```

Comment:
`interfaces` are not directly under `spec`. They are under `spec.domain.devices`.